index.js

```diff
@@ -41,3 +41,77 @@ app.get('/dogs', (req, res) => {
         })
     })
 })
+
+app.post('/dogs/:id/win', jsonParser, (req, res) => {
+    let id = req.param('id')
```

Comment:
I get the error `express deprecated req.param(name): Use req.params, req.body, or req.query instead index.js:46:18`

Reply:
done